sdb/commands/internal/util.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Any, Callable, Iterable, Optional
+
+import drgn
+
+
+def sort_objects_by(objs: Iterable[drgn.Object],
+                    field_getter: Callable[[drgn.Object], Any],
+                    reverse_sort: Optional[bool] = False
+                   ) -> Iterable[drgn.Object]:
+    obj_list = [(field_getter(obj), obj) for obj in objs]
+    obj_list.sort(reverse=reverse_sort)
+    for _, obj in obj_list:
+        yield obj
```

Comment:
I feel like this function is unnecessary, as it's just re-implementing the `sorted` builtin ([link](https://docs.python.org/3/library/functions.html#sorted).

Couldn't consumers more simply do this:
```
            yield from sorted(
                self.__no_input_iterator(),
                key=Slabs.FIELDS[self.args.s],
                reverse=(
                    self.args.s not in Slabs.DEFAULT_INCREASING_ORDER_FIELDS))
```
?

Reply:
Definitely! Applying the change now